aws/ca/.terraform.lock.hcl

```diff
@@ -0,0 +1,59 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Shouldn't be committed?

Reply:
https://github.com/scalar-labs/scalar-terraform-examples/pull/13#discussion_r585217817